src/main/java/com/jcraft/jsch/OpenSSHConfig.java

```diff
@@ -223,6 +239,24 @@ private String find(String key) {
         }
       }
       */
+
+      if (keysWithListAdoption.contains(key) && value != null && (value.startsWith("+") || value.startsWith("-") || value.startsWith("^"))) {
+
+        String origConfig = JSch.getConfig(originalKey).trim();
+
+        if (value.startsWith("+")) {
+          value=origConfig + "," + value.substring(1);
+        } else if (value.startsWith("-")) {
+          List<String> algList = Arrays.stream(origConfig.split(",")).collect(Collectors.toList());
```

Comment:
Should this use `Util.split(origConfig, ",")`?

Reply:
yes, I will change it